src/app/components/auth/login/login.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="flex justify-center items-center h-full">
+<!-- <div class="flex justify-center items-center h-full">
```

Comment:
Pode remover os códigos comentados quando for submeter, a não ser que seja algo importante